### PR TITLE
feat:wishlist page done

### PR DESCRIPTION
**Pull request by Developers-codz**
Created responsive wishlist page
Added header in cart page as well

Do review my PR and give valuable feedbacks. Thank you :)

### CODE REVIEW

File: styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+.item-price{
+    color:var(--primary)
+}
+.move-to-cart-btn-wrapper{
+    margin:1rem 0rem;
+    width: inherit;
+}
+.btn-to-cart{
+    width:inherit;
+    padding:.5rem 3rem .5rem 3rem;
```
Comment (partha8) [inline styles/wishlist.css]: I think you missed this here, padding: .5rem .3rem;

**Review by partha8 — APPROVED** [review]
Everything looks good, but there's problem with indentation in your HTML file.

Great work on the UI btw.

**Comment by Developers-codz** [comment]
> Everything looks good, but there's problem with indentation in your HTML file.
> 
> Great work on the UI btw.

ya ya fixing it. Thanks for reviewing :)

**Review by aayushjain-code — COMMENTED** [review]
Everything looks fine to try to add some padding in the navbar from left and right.
Also in the banner.

**Review by arjun-computer-geek — APPROVED** [review]
Looks good. Ready to merge 😁

**Comment by Developers-codz** [comment]
> Everything looks fine to try to add some padding in the navbar from left and right. Also in the banner.

I think its looking fine. Thank you for reviewing by the way :)

**Pull request merged**